cmd/server/main.go

```diff
@@ -387,6 +392,11 @@ func (p *Proxy) runMasterServer(ctx context.Context, o *ProxyRunOptions, server
 }
 
 func (p *Proxy) runUDSMasterServer(ctx context.Context, o *ProxyRunOptions, s *server.ProxyServer) (StopFunc, error) {
+	if o.deleteUDSFile {
+		if err := os.Remove(o.udsName); err != nil {
```

Comment:
I believe this will error if the file does not exist. (Eg. first time run). It seems like we should only do this if the file exists. Given that maybe check if the fix exists.  If it does and deleteUDSFile is set then delete, if deleteUDSFile is not set give a detailed error message of the problem.

Reply:
I updated the code to ignore the error if it's a "file not exist" error to avoid printing the redundant warning message.

>  if deleteUDSFile is not set give a detailed error message of the problem

Current error message is: "failed to run the master server: failed to listen(unix) name ./<udsName>: listen unix ./<udsName>: bind: address already in use". I think it's clear enough.